css/src/components/input.css

```diff
@@ -128,6 +128,30 @@
   color: var(--inverse-lighter);
 }
 
+.Input-icon--right:focus-visible .Icon {
+  outline: var(--spacing-s) solid var(--secondary-shadow);
+  border-radius: 10px;
+}
+
+.Icon:focus-visible {
+  outline: var(--spacing-s) solid var(--secondary-shadow);
+  border-radius: 10px;
+}
+
+.Input-icon--right:hover .Icon {
```

Comment:
Why are using child-parent selectors here? Can we do this by passing CSS to the parent class only?